parcel-bundler/parcel
# Invalidate assets built with babel on BABEL_ENV and NODE_ENV change

Related: #4438 #4432

* Removes the references to `process.env` introduced with #4438 as Parcel's own environment is used for `options.env`.
* Always invalidates assets built with the Babel transformer on the BABEL_ENV and NODE_ENV environment variables. **Note**: This should probably eventually move to the config object.

cc @garthenweb 

Test Plan: 
* Updated integration tests
* Use the following `.babelrc` in the simple example:
```
{
  "env": {
    "foo": {
      "plugins": ["babel-plugin-console-source"]
    }
  }
}
```
* Modify `yarn demo` modified to do `parcel build`
* Run `yarn demo` and verify the console.log is unmodified
* Without clearing the cache, run `BABEL_ENV=foo yarn demo` and verify the console.log is modified.
